packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -311,6 +340,32 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
     return renderedItems;
   }
 
+  private _renderSectionItem(item: IContextualMenuItem, index: number, hasCheckmarks: boolean, hasIcons: boolean) {
+    const section = item.sectionProps;
+    if (!section) {
+      return;
+    }
+
+    let headerItem = section.title ? this._renderHeaderMenuItem(section.title, index, hasCheckmarks, hasIcons) : undefined;
+
+    if (section.items && section.items.length > 0) {
+      return <li
+        role='presentation'
+        key={ section.key }>
+        <div role='group'>
+          <ul className={ css('ms-ContextualMenu-list is-open', styles.list) }>
+            { section.topDivider ? this._renderSeparator(index, undefined, true, true) : undefined }
```

Comment:
typically in react, you do this with section.topDivider && this._renderSeparator instead of a ternary. Does that work here?